src/themes/SciReactUIProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { BrowserRouter } from "react-router-dom";
+import { ThemeProvider, ThemeProviderProps } from "./ThemeProvider";
+
+type SciReactUIProviderProps = React.PropsWithChildren<ThemeProviderProps>;
```

Comment:
Should we also add `BrowserRouterProps` to the type so we can add them with `<BrowserRouter{...props.router}>` or similar?